examples/kubernetes-client-examples/README.MD

```diff
@@ -0,0 +1,48 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Flink Kubernetes Client Code Example
+
+## Overview
+
+This is an end-to-end example of running Flink Job via code using the Flink Kubernetes Operator.
+
+It is only intended to serve as a showcase of how Flink Operator classes can be used to submit flink job the Flink Kubernetes Operator.
+
+
+*What's in this example?*
+
+1. Code approach to the simple basic.yaml
```

Comment:
wording: Sample code for submitting an application similar to `examples/basic.yaml` programatically.